lib/beaker-hostgenerator/data.rb

```diff
@@ -185,6 +185,17 @@ def osinfo
                           'box' => 'debian/bullseye64',
                         },
                       },
+                      'debian11-AARCH64' => {
+                        general: {
+                          'platform' => 'debian-11-aarch64',
+                        },
+                        abs: {
+                          'template' => 'debian-11-arm64',
+                        },
+                        vmpooler: {
+                          'template' => 'debian-11-arm64',
+                        },
```

Comment:
Perhaps enhance this part to generate it:
https://github.com/voxpupuli/beaker-hostgenerator/blob/f47bca25f12c53161c923cdda7c8f8462b9f4594/lib/beaker-hostgenerator/hypervisor/vmpooler.rb#L25-L34

Reply:
Actually, I don't think we need to use the vmpooler hypervisor moving forward.

Reply:
I'm also happy to drop vmpooler, or at least stop updating it.